examples/parallel.rs

```diff
@@ -0,0 +1,356 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+use cupcake::traits::*;
+use cupcake::DefaultShemeType;
+use cupcake::FVCiphertext;
+use cupcake::FVPlaintext;
+use cupcake::integer_arith::scalar::Scalar;
+use cupcake::SecretKey;
+
+use rayon::iter::IntoParallelIterator;
+use rayon::iter::ParallelIterator;
+use rayon::iter::IndexedParallelIterator;
+
+use serde::{Deserialize, Serialize};
+use std::fmt::{Debug, Display, Error, Formatter};
+
```

Comment:
`ByteBuffer` is already serialized as a `vec<u8>` so it shouldn't need a `Serialize` or `Deserialize` trait